### PR TITLE
[ADD] backport override_mail_recipients

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: override_mail_recipients/models/ir_mail_server.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models
```
Comment (NL66278) [inline override_mail_recipients/models/ir_mail_server.py]: In 7.0?

File: override_mail_recipients/models/ir_mail_server.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+from email.Utils import COMMASPACE
+from openerp.addons.base.ir.ir_mail_server import extract_rfc2822_addresses
+
+
+class IrMailServer(models.Model):
```
Comment (NL66278) [inline override_mail_recipients/models/ir_mail_server.py:26]: In 7.0?

Comment (hbrunn) [reply override_mail_recipients/models/ir_mail_server.py:26]: now yes, I forgot to push the actually relevant commit...

**Comment by NL66278** [comment]
:+1:

**Pull request merged**